### ISSUE TITLE
Basket order of StopLossLimit orders is placing with price 0

### ISSUE THREAD

**Issue by dheeruasdf** [krishnavelu/alice_blue#345]
Hi @krishnavelu,

When i am trying to place an order of StopLossLimit, then the order is being placed with price 0.0 value and it is not respecting the value I have given.

**Comment by krishnavelu** [comment]
Read [this](https://github.com/krishnavelu/alice_blue#read-this-before-creating-an-issue) and follow before creating a new issue...

**Comment by tdheeraj333** [comment]
This is the sample code which am using
```
def place_basket_order():
    print('placed basket order')
    order1 = {"instrument": ce_script,
              "order_type": OrderType.StopLossLimit,
              "quantity": quantity,
              "price": ce_ltp - 10,
              "trigger_price": ce_ltp,
              "transaction_type": TransactionType.Sell,
              "product_type": ProductType.Intraday}
    order2 = {"instrument": pe_script,
              "order_type": OrderType.StopLossLimit,
              "quantity": quantity,
              "price": pe_ltp - 10,
              "trigger_price": pe_ltp,
              "transaction_type": TransactionType.Sell,
              "product_type": ProductType.Intraday}
    orders = [order1, order2]
    return alice.place_basket_order(orders)
```

Here although I have passed the price, the order is being placed with price 0.0
![image](https://user-images.githubusercontent.com/11541920/135273615-d9fba139-57c7-41e8-9db3-331db358f481.png)

**Comment by dheeruasdf** [comment]
This is the sample code which am using
```
def place_basket_order():
    print('placed basket order')
    order1 = {"instrument": ce_script,
              "order_type": OrderType.StopLossLimit,
              "quantity": quantity,
              "price": ce_ltp - 10,
              "trigger_price": ce_ltp,
              "transaction_type": TransactionType.Sell,
              "product_type": ProductType.Intraday}
    order2 = {"instrument": pe_script,
              "order_type": OrderType.StopLossLimit,
              "quantity": quantity,
              "price": pe_ltp - 10,
              "trigger_price": pe_ltp,
              "transaction_type": TransactionType.Sell,
              "product_type": ProductType.Intraday}
    orders = [order1, order2]
    return alice.place_basket_order(orders)
```

Here although I have passed the price, the order is being placed with price 0.0
![image](https://user-images.githubusercontent.com/11541920/135273615-d9fba139-57c7-41e8-9db3-331db358f481.png)

**Comment by krishnavelu** [comment]
The error message shown above and the fix you've done and the problem you are facing, doesn't have any relationship with your fix. Also it shows multiple merge pull request in the PR.

![image](https://user-images.githubusercontent.com/38440742/135721287-9662ef3b-19c9-40dc-a202-7b9eb6bd958d.png)

**Comment by tdheeraj333** [comment]
Hi @krishnavelu ,

The problem is recently NSE has disabled `Stoploss Market` for F&O orders and thus when i place an order with trigger price and no price i.e; `price : 0.0` it is failing with that error.

This is the explanation I have received from alice_blue.

and when I have checked the code I have noticed that the if logic of checking `Order.StoplossLimit`  comes after the `Order.Limit` but as I am placing an order for `Order.StoplossLimit` it is going to the else block of `Order.Limit` and making the `price to 0.0`

I have tested this change in my local and its working fine.

I am sorry about the commits. I didn't noticed the other commits.
I shall send the updated pull request.
